xla/debug_options_flags.cc

```diff
@@ -159,6 +159,7 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
   opts.set_xla_gpu_nccl_termination_timeout_seconds(-1);
   opts.set_xla_gpu_enable_shared_constants(true);
   opts.set_xla_gpu_enable_nccl_user_buffers(false);
+  opts.set_xla_gpu_enable_nvshmem(true);
```

Comment:
Can you rename the flag to `xla_gpu_experimental_enable_nvshmem` as we aim to enable this by default at some point